src/main/java/vectorwing/farmersdelight/common/Configuration.java

```diff
@@ -75,6 +78,11 @@
 				.defineList("canvasSignDarkBackgroundList", ImmutableList.of("gray", "purple", "blue", "brown", "green", "red", "black"), obj -> true);
 		COMMON_BUILDER.pop();
 
+		COMMON_BUILDER.comment("Recipe book").push(CATEGORY_RECIPE_BOOK);
+		ENABLE_RECIPE_BOOK_COOKING_POT = COMMON_BUILDER.comment("Should the Cooking Pot have a Recipe Book available on its interface?")
+				.define("enableRecipeBookCookingPot", true);
+		COMMON_BUILDER.pop();
```

Comment:
TODO: Study a feature for automatically unlocking all cooking recipes, if a config is enabled (would be disabled by default).

Reply:
Marked for later consideration.